solarFlux/fluxes.rml

```diff
@@ -87,5 +87,14 @@
 
 		<parameter name="seed" value="137" />
 	</TRestAxionSolarQCDFlux>
+
+	<TRestAxionSolarHiddenPhotonFlux name="OSheaHiddenPhoton" verboseLevel=="warning" >
+		<parameter name="couplingType" value="g_eff"/>
+		<parameter name="couplingStrength" value="1"/>
+		<parameter name="fluxDataFile" value="hiddenPhoton_flux_OShea_202306.dat"/>
```

Comment:
I thought you will need 3 files?

Reply:
Yes the 3 that I added into the hidden photon folder. Can I just call them whatever I want in the parameter name?